dali/operators/audio/resample.cc

```diff
@@ -96,6 +96,17 @@ type. Example::
 )",
     nullptr, false);
 
+// Deprecated alias
+DALI_SCHEMA(experimental__AudioResample)
```

Comment:
You need some 1-line docstring so it shows in the table of contents
.DocStr("Legacy alias for operator introduced as :meth:`audio_resample`.")